src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import java.lang.reflect.Array;
+import java.util.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            String nextItem = s.nextLine();
+            loadItem(nextItem);
+            System.out.println(nextItem);
+        }
+    }
+
+    private static void loadItem(String item) {
+        char type = item.charAt(1);
+        Task task;
+        String description;
+        switch (type){
+        case 'T':
```

Comment:
Using single characters for task type could be a little vague. Consider how you could improve code readability here.